patches/node/chore_prevent_warn_non_context-aware_native_modules_being_loaded.patch

```diff
@@ -46,96 +46,3 @@ index 541c18d8032c5247433367f9e04c946cf203d475..b04e3f1fc56a86abe18f924e4d9b7fb5
        if (process.argv[1]) {
          try {
            resolvedArgv = Module._resolveFilename(process.argv[1], null, false);
-diff --git a/src/env-inl.h b/src/env-inl.h
-index c1853f81b68bd22d20fb99877f4c500a384e9545..623e9d4e429c03bb267539a318166f3ef3b9c501 100644
---- a/src/env-inl.h
-+++ b/src/env-inl.h
-@@ -550,6 +550,14 @@ inline bool Environment::force_context_aware() const {
-   return options_->force_context_aware;
- }
- 
-+inline void Environment::set_warn_context_aware(bool value) {
-+  options_->warn_context_aware = value;
-+}
-+
-+inline bool Environment::warn_context_aware() const {
-+  return options_->warn_context_aware;
-+}
-+
- inline void Environment::set_abort_on_uncaught_exception(bool value) {
-   options_->abort_on_uncaught_exception = value;
- }
-diff --git a/src/env.h b/src/env.h
-index f89365a1aa7ffacbb423e01a68f484992751f76f..38d17f4e18aa38fde2c2f59a9816c8fb0f65fd51 100644
---- a/src/env.h
-+++ b/src/env.h
-@@ -949,6 +949,8 @@ class Environment : public MemoryRetainer {
- 
-   inline void set_force_context_aware(bool value);
-   inline bool force_context_aware() const;
-+  inline void set_warn_context_aware(bool value);
-+  inline bool warn_context_aware() const;
- 
-   // This stores whether the --abort-on-uncaught-exception flag was passed
-   // to Node.
-diff --git a/src/node_binding.cc b/src/node_binding.cc
-index ca5a01f925a2ae69ba4295d82316e546f45c60cd..f85ab2332a1c0267bd50d5f979d90e55c84a2990 100644
---- a/src/node_binding.cc
-+++ b/src/node_binding.cc
-@@ -3,6 +3,7 @@
- #include <atomic>
- #include "env-inl.h"
- #include "node_native_module_env.h"
-+#include "node_process.h"
- #include "util.h"
- 
- #if HAVE_OPENSSL
-@@ -463,8 +464,20 @@ void DLOpen(const FunctionCallbackInfo<Value>& args) {
-       if (mp->nm_context_register_func == nullptr) {
-         if (env->force_context_aware()) {
-           dlib->Close();
--          THROW_ERR_NON_CONTEXT_AWARE_DISABLED(env);
-+          char errmsg[1024];
-+          snprintf(errmsg,
-+                   sizeof(errmsg),
-+                   "Loading non-context-aware native module in renderer: '%s', but app.allowRendererProcessReuse is true. See https://github.com/electron/electron/issues/18397.",
-+                   *filename);
-+          env->ThrowError(errmsg);
```

Comment:
I think this part of the patch should probably stick around, it provides substantially more information / context than the built-in nodejs error message.  Perhaps modified to remove the reference to `app.allowRendererProcessReuse` but the github issue link is very useful IMO